sympy/solvers/solvers.py

```diff
@@ -3102,7 +3166,7 @@ def _invert(eq, *symbols, **kwargs):
     (0, sqrt(x) + y)
 
     If there is more than one symbol in a power's base and the exponent
-    is not an Integer, then the principal root will be used for the
+    is not an integer, then the principal root will be used for the
```

Comment:
I think uppercase I was correct here, as it was referring to the class sympy.Integer.